ansible/roles/uan_hardening/tasks/disable_ssh_out_nmn_to_management_ncns.yml

```diff
@@ -0,0 +1,58 @@
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+- set_fact:
+    nmn_subnet_network_from_uan: "{{ ansible_nmn0['ipv4']['network'] }}" #get subnet cidr
+
+- set_fact:
+    nmn_subnet_mask_from_uan: "{{ ansible_nmn0['ipv4']['netmask'] }}"
+
+#Set variable nmn_network_cidr to the calculated CIDR of the NMN
+# network by getting subnet and subnet mask from ansible inventory
+# from the two set_facts directly above.
+- set_fact:
+    nmn_network_cidr: "{{ (nmn_subnet_network_from_uan + '/' + nmn_subnet_mask_from_uan) | ipaddr('network/prefix') }}"
+
+#Add iptables drop rule, for outbound only traffic, only out the nmn0
+# UAN interface, and only for destination TCP port 22 ssh traffic.
+# Does not affect inbound traffic.
+- name: Block ssh to NCN IPs from UAN NMN$
+  iptables:
+    chain: OUTPUT
+    jump: DROP
+    protocol: tcp
+    out_interface: nmn0
+    destination_port: 22
+    destination: "{{ nmn_network_cidr }}"
```

Comment:
You will also need to block the NMNLB network.   An SSH to any of those IPs will take you to one of the worker nodes.

For example:
```
uan01:~ # ssh root@10.92.100.71
Password: 
Last login: Mon Jun 27 00:42:53 2022 from 10.252.1.21
ncn-w002:~ # 
{noformat}